### ISSUE TITLE
DisableLinterReason not catching a comment for an inline disable

### ISSUE THREAD

**Issue by connorshea** [brigade/scss-lint#749]
I have the following SCSS:

```scss
// Unfortunately this selector is necessary for CSS specificity reasons.
> .notifications-container ul > li:last-of-type { // scss-lint:disable SelectorDepth
     border-bottom: 0;
}
```

I remember fixing the SCSS linter reporting this unfortunately necessary selector with the above. Adding it as a comment at the end of a line was documented somewhere, but I can't find it now. It's supposed to disable the rule for only that line.

The `DisableLinterReason` rule is throwing a warning despite the fact that I have a comment explanation up above.

**Comment by connorshea** [comment]
Also happens with this bit of code:
```scss
// Test
-webkit-user-select: none; // scss-lint:disable VendorPrefix
```

**Comment by sds** [comment]
Fixed by #762.
